server/http.go

```diff
@@ -175,7 +175,9 @@ func (s *HTTPd) RegisterSiteHandlers(site site.API, valueChan chan<- util.Param)
 			"vehicleDetect":    {"PATCH", "/vehicle", vehicleDetectHandler(lp)},
 			"remotedemand":     {"POST", "/remotedemand/{demand:[a-z]+}/{source:[0-9a-zA-Z_-]+}", remoteDemandHandler(lp)},
 			"enableThreshold":  {"POST", "/enable/threshold/{value:-?[0-9.]+}", floatHandler(pass(lp.SetEnableThreshold), lp.GetEnableThreshold)},
+			"enableDelay":      {"POST", "/enable/delay/{value:[0-9smh]+}", durationHandler(pass(lp.SetEnableDelay), lp.GetEnableDelay)},
 			"disableThreshold": {"POST", "/disable/threshold/{value:-?[0-9.]+}", floatHandler(pass(lp.SetDisableThreshold), lp.GetDisableThreshold)},
+			"disableDelay":     {"POST", "/disable/delay/{value:[0-9smh]+}", durationHandler(pass(lp.SetDisableDelay), lp.GetDisableDelay)},
```

Comment:
Lass uns hier nicht das proprietäre Go-Durationformat nutzen und lieber auf Sekunden gehen. Die sollten in ganzzahlig vollkommen ausreichend sein und machen die API einfacher.